#Parameter server configuration for bta_ros.

#udpControlOutIpAddr: 
#udpControlOutIpAddrLen:
#udpControlOutPort:
#udpControlInIpAddr:
#udpControlInIpAddrLen:
#udpControlInPort: 
#tcpDataPort:
#uartPortName:
#uartBaudRate:
#uartDataBits:
#uartStopBits:
#uartParity:
#uartTransmitterAddress:
#uartReceiverAddress
#serialNumber:
#calibFileName: 
#zFactorsFileName: 
#wigglingFileName: 
frameMode: 6  # 2-DistAmpFlags // 4-XYZAmp // 6-XYZAmpFlags
#infoEvent:
#frameArrived:
#frameQueueMode:
verbosity: 5
#frameQueueLength: 1

# Optional parameter.
#frameRate: 15
#integrationTime: 1500

#pll0:
#pll1:
#ampTreshold:
#saturationTreshold:
